src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java

```diff
@@ -71,50 +71,94 @@ public void init(FMLInitializationEvent event) {
+		}
 
 		// Sink
-		if(CookingConfig.enableSink) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockSink), "III", "WBW", "WWW", 'I', "ingotIron", 'W', "logWood", 'B', Items.water_bucket));
+		if (CookingConfig.enableSink) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockSink), "III", "WBW", "WWW", 'I', "plateIron", 'W', "plateWood", 'B', Items.water_bucket));
+			} else {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockSink), "III", "WBW", "WWW", 'I', "ingotIron", 'W', "logWood", 'B', Items.water_bucket));
+			}
 		}
 
 		// Toaster
-		if(CookingConfig.enableToaster) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockToaster), "   ", "IRI", "SSS", 'I', "ingotIron", 'R', "dustRedstone", 'S', "stoneSmooth"));
+		if (CookingConfig.enableToaster) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockToaster), "IBI", "IRI", "SSS", 'B', Blocks.iron_bars, 'I', "plateIron", 'R', "springIron", 'S', "plateStone"));
+			} else {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockToaster), "   ", "IRI", "SSS", 'I', "ingotIron", 'R', "dustRedstone", 'S', "stoneSmooth"));
+			}
 		}
 
 		// Cooking Table
-		if(CookingConfig.enableCraftingBook) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockCookingTable), "CCC", "WBW", "WWW", 'B', new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), 'W', "logWood", 'C', new ItemStack(Blocks.stained_hardened_clay, 1, 15)));
+		if (CookingConfig.enableCraftingBook) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockCookingTable), "CCC", "WBW", "SWS", 'B', new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), 'W', "plateWood", 'C', "plateStone", 'S', "screwIron"));
+			} else {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockCookingTable), "CCC", "WBW", "WWW", 'B', new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), 'W', "logWood", 'C', new ItemStack(Blocks.stained_hardened_clay, 1, 15)));
+			}
 		}
-
+		
 		// Cooking Oven
-		if(CookingConfig.enableOven) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockOven), "GGG", "IFI", "III", 'G', new ItemStack(Blocks.stained_glass, 1, 15), 'I', "ingotIron", 'F', Blocks.furnace));
+		if (CookingConfig.enableOven) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockOven), "GGG", "IFI", "III", 'G', new ItemStack(Blocks.stained_glass_pane, 1, 15), 'I', "plateIron", 'F', "craftingIronFurnace"));
```

Comment:
Are stained glass and iron furnaces possible pre-steam? I'm not sure if they are.

Reply:
why it must be avaialable pre steam ? Yea have much todo with other stuff. Planting stuff is always steam age in my playthrough

Reply:
thanks for your review but lets hear mitch on this as well. And as a pack creator i guess my ideas and words counts too?